README.md

```diff
@@ -1,33 +1,68 @@
 <div align="center">
-<img src="./docs/assets/logo.png" alt="Margarita" title="Margarita" width="300"/>
+  <a href="http://margarita.kiwi.com"><img src="./docs/assets/logo.png" alt="Margarita" title="Margarita" width="200" /></a>
 </div>
+<h1 align="center">Margarita</h1>
+<h4 align="center">Multi-platform <span title="travel">✈️</span> travel app  for iOS, Android and Web<br></h4>
 
-# Margarita
+ <p align="center">built with <strong>React Native, GraphQL and <a href="https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/" target="_blank">Tequila API</a></strong></p>
 
-> Showcasing the power of [Tequila](https://partners.kiwi.com/presenting-tequila-revolution-travel-industry/)
+## Demo
 
-This project is a living example of a client app you could start building with Tequila.
+Live demo: https://kiwicom-margarita.netlify.com (web version)
 
-- 🕸️ Web version: https://kiwicom-margarita.netlify.com
-- 🎮 GraphQL Playground https://p2kwd3i3a8.execute-api.eu-central-1.amazonaws.com/staging/graphql
-- 🚀 GraphQL Voyager: https://margarita-graphql-voyager.now.sh/
+**If you like this project**, please give us ⭐ on GitHub — it helps!
 
 <div align="center">
-<img src="./docs/assets/search.png" alt="Search" title="Search" width="200"/>
-<img src="./docs/assets/results.png" alt="Results" title="Results" width="200"/>
-<img src="./docs/assets/bookings.png" alt="Bookings" title="Bookings" width="200"/>
+  <img src="./docs/assets/screens/search.png" alt="Search" title="Search" width="250"/>
+  <img src="./docs/assets/screens/results.png" alt="Results" title="Results" width="250"/>
+  <img src="./docs/assets/screens/mmb.png" alt="Bookings" title="Bookings" width="250"/>
 </div>
 
-## 📖 Documentation
+<img src="./docs/assets/mac.png" alt="Search" title="Search" />
 
-The full documentation for **Margarita** can be found on our [website](https://kiwicom.github.io/margarita/docs/).
+## Devstack
 
-## 💻 Developing
+This project follows react-native-web philosophy of **Write Once, Render Anywhere**.
 
-- [Getting Started](https://kiwicom.github.io/margarita/docs/getting-started)
-- [Technical Overview](https://kiwicom.github.io/margarita/docs/technical-overview)
+- React Native
+- Expo
+- React Native Web
+- Next.js
+- Relay
+- Flow
+
+More detailed info about our devstack can be found in [Technical Overview docs](https://margarita.kiwi.com).
```

Comment:
One more link change. Should be https://kiwicom.github.io/margarita/docs/technical-overview

Reply:
Done